spring-kafka/src/main/java/org/springframework/kafka/listener/KafkaMessageListenerContainer.java

```diff
@@ -299,7 +299,7 @@ public Map<String, Collection<TopicPartition>> getAssignmentsByClientId() {
 
 	@Override
 	public boolean isContainerPaused() {
-		return isPaused() && this.listenerConsumer != null && this.listenerConsumer.isConsumerPaused();
+		return isPauseRequested() && this.listenerConsumer != null && this.listenerConsumer.isConsumerPaused();
```

Comment:
I would think that `isPaused` is more appropriate here. Thus, we need to reevaluate the removal of `isPaused` from the container API. What is your reasoning for this change?